src/response.rs

```diff
@@ -263,6 +263,14 @@ impl Into<http::Response> for Response {
     }
 }
 
+impl From<serde_json::Value> for Response {
```

Comment:
We should consider adding this impl to `Body` as well!

Reply:
What if instead of two different From<serde_json::Value>'s:
`impl From<serde_json::Value> for Body`
and
a `impl<T: Into<Body>> From<T> for Response` with `Response::new(StatusCode::Ok).set_body(self)`

Reply:
Does that still allow `Ok(json!({}))`? It certainly sounds more flexible.

Reply:
@Fishrock123 It would, _if_ I could figure out how to make it work with coherence rules. http_types::Response [has something like this](https://github.com/http-rs/http-types/blob/master/src/response.rs#L609-L619)

Reply:
@jbr I think it's possible to do that, but believe there was a reason why we shouldn't do that in Tide. I forget what that was though. I think if we can add the Body impl manually we'll still be good, and we can perhaps explore generalizing things in a future PR.